settings.gradle

```diff
@@ -13,6 +13,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+plugins {
+    id 'com.gradle.enterprise' version '3.13.1'
```

Comment:
Would these plugins be loaded for local builds as well (I understand that they won't be used based on code below)?